graspologic/plot/plot.py

```diff
@@ -97,8 +100,10 @@ def _transform(arr, method):
             transformer = Binarizer().fit(arr)
             arr = transformer.transform(arr)
         else:
-            msg = f"Transform must be one of {{log, log10, binarize, zero-boost, \
-            simple-all, simple-nonzero}}, not {method}."
+            msg = "Transform must be one of {log, log10, binarize, zero-boost, simple-all, \
+            simple-nonzero, not {}.".format(
+                method
+            )
```

Comment:
why this change?

Reply:
I never changed this? @bdpedigo ... what should i do?

Reply:
i think a common thread is these are all changes that happened in a recent PR, and somehow when you rebased/merged from `dev`, you probably chose some option that reverted these back to what they used to be